src/main/main.ts

```diff
@@ -90,10 +94,30 @@ const createWindow = async () => {
     } else {
       mainWindow.show();
     }
+
+    pyServer = startServer();
+
+    if (pyServer !== null && pyServer.stderr !== null) {
+      pyServer.stderr.once('data', (data) => {
```

Comment:
Is there a way to get this logging to stdout instead of stderr? Using stderr seems like a bit of a hack

Reply:
The flask app logs to both stdout and stderr.

This gets logged to stdout
```
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
```
This gets logged to stderr
```
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
```

It should ping the server once flask displays this `Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)` since that indicates that the server is now up and running.

Flask also logs out the requests it gets to stderr.

If I where to change it to log everything to stdout then I would have to change the event listner from `once` to `on` (since it will no longer be the first thing that is logged) and only execute the health check if `Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)` is part of what is logged. 

It would also cause the event listener to fire off unnecessarily for every request it recieves.

Reply:
Okay, sure - nice analysis. Perhaps add a comment as a quick explainer so this doesn't come up again

Reply:
Okay, Im going to link to this discussion.